app/cp_nigeria/helpers.py

```diff
@@ -119,24 +126,145 @@ def __init__(self):
         self.doc = Document()
         self.logo_path = "static/assets/logos/cpnigeria-logo.png"
 
+        self.text_parameters = dict(
```

Comment:
This is meant to be populated with the project handle, maybe use a method of the `Project` model to return this dict